### ISSUE TITLE
Thought out loud - Chrome Extension

### ISSUE THREAD

**Issue by flexchar** [antonmedv/finder#80]
This would make into a wonderful Chrome Extension :)

**Comment by rxliuli** [comment]
Yes, I plan to create a Chrome extension for this, but after some initial testing, I found issues with the generated CSS. For example, it cannot handle the following scenarios:

1. No automatic handling of aria-label, such as `[aria-label="Profile"]` or `button[aria-label="Grok actions"]`
2. Cannot handle scenarios with SVG elements in child elements, where `:has(> [d="M5.651 19h12.698c"])` could be used to generate stable CSS
3. Ignores `data-testid`, which is almost synonymous with stable matching, like `[data-testid="BottomBar"]`
4. Ignores href attributes, which can make stable CSS matching easier, for example `[data-testid="tweet"]:has(a[href*="utm_source=twitter"][href*="twclid"])`
5. Given the prevalence of LLMs, perhaps we could use LLM to analyze and generate CSS selectors

**Comment by rxliuli** [comment]
Published: https://chromewebstore.google.com/detail/css-selector-generator/bgijemefjbbcbcbjigojegjbfdlfigpg
